src/scripts/main.js

```diff
@@ -1,7 +1,32 @@
 'use strict';
 
+function createElement(tagName, text) {
+  const element = document.createElement(tagName);
+
+  element.textContent = text;
+
+  return element;
+}
+
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const html = document.documentElement;
+  const notification = document.createElement('div');
+
+  const notificationTitle = createElement('h2', title);
+  const notificationDescription = createElement('p', description);
+
+  notificationTitle.classList.add('title');
+  notification.classList.add('notification', type);
+
+  notification.append(notificationTitle);
+  notification.append(notificationDescription);
+
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  html.append(notification);
+
+  setTimeout(() => (notification.style.display = 'none'), 2000);
```

Comment:
The current logic for hiding the notification using `setTimeout` only sets the `display` style to 'none'. This does not remove the notification from the DOM, which might lead to unnecessary elements being kept in the document. Consider using `notification.remove()` instead to completely remove the element from the DOM after the timeout.